python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -786,6 +792,82 @@ def maskSaturatedPixels(self, badAmpDict, ccdExposure, detector, detectorConfig,
                 badAmpDict[ampName] = True
                 maskView |= maskView.getPlaneBitMask("BAD")
 
+        # Edge bleed masking
+        if self.config.doITLEdgeBleedMask and detector.getPhysicalType() == 'ITL':
+
+            # Get footprint of saturated pixels
+            thresh = afwDetection.Threshold(limits[self.config.saturatedMaskName])
+            fpList = afwDetection.FootprintSet(maskedImage, thresh).getFootprints()
+
+            # Go through footprints to select saturated cores
+            for i in range(len(fpList)):
```

Comment:
I worry about the speed of this for loop ... though it may be fine.  But maybe it would be easier to pull out all the areas from fpList and then numpy check the array for those over 10000?

e.g.
```
areas = numpy.asarray([fp.getArea() for fp in fpList])
large, = numpy.where(areas >= minArea)
if len(large) == 0:
    return
```